.scripts/util.sh

```diff
@@ -59,25 +63,66 @@ is_generated() {
     fi
 }
 
+select_browser_caller() {
+    if [[ "$1" = "chromium" || "$BROWSER" = "chromium" ]]; then
+        BROWSER="chromium"
+        SYS_OPTS="\
+        -DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/include\
+        -DBROWSER_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/x64\
+        -DBROWSER=$BROWSER"
+    elif [[ "$1" = "mswebview2" || "$BROWSER" = "mswebview2" ]]; then
+        BROWSER="mswebview2"
+        SYS_OPTS="\
+        -DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_NUGET_V/build/native/include\
+        -DBROWSER_LIB_DIR=$PACKAGE_DIR/Microsoft.Web.WebView2.$MSWEBVIEW_NUGET_V/build/native/x64\
+        -DBROWSER=$BROWSER"
+    else
+        # Handle both empty and invalid input in one condition
+        throw_error "Invalid input: $1. Resorting to default browser: Chromium"
+        BROWSER="chromium"
+        SYS_OPTS="-DBROWSER_INCLUDE_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/include -DBROWSER_LIB_DIR=$PACKAGE_DIR/chromiumembeddedframework.runtime.$CHROMIUM_NUGET_V/build/native/x64 -DBROWSER=$BROWSER"
```

Comment:
If we reach here, the user has given an invalid option, so we notify them, show them the valid options, and recurse the function to set the default:
```bash
else
    echo "'$1' was not a valid browser option: $BROWSER_OPTIONS"
    select_browser_caller
fi
```

Follow the logic back to the top of this function:
- $1 will be undefined
- The user will be messaged that the `chromium` option was set (so no need to do it at the bottom).
- `SYS_OPTS` will be correctly set

Do not Repeat Yourself. This is a prime location for little gremlins to creep in if we ever change the `-DBROWSER_*` setting logic and forget about the conditional block at the end...